packages/cli/test/integration/generators/belongsto.relation.integration.js

```diff
@@ -239,9 +235,9 @@ context('check if the controller file created ', () => {
     /constructor\(\n    \@repository\(OrderClassTypeRepository\)\n    public orderClassTypeRepository: OrderClassTypeRepository,\n  \) \{ \}\n/,
   ];
   const indexExport = [
-    /export \* from '.\/order-customer.controller';/,
-    /export \* from '.\/order-class-customer-class.controller';/,
-    /export \* from '.\/order-class-type-customer-class-type.controller';/,
+    /export \* from ".\/order-customer.controller";/,
+    /export \* from ".\/order-class-customer-class.controller";/,
+    /export \* from ".\/order-class-type-customer-class-type.controller";/,
```

Comment:
Hmm, looks like the double-quote characters have sneaked back with Prettier gone :(

Could you please check what formatting options are offered by ts-morph to see if we can tell it to use single quotes and 2-space indentation?

- https://dsherret.github.io/ts-morph/manipulation/formatting
- https://github.com/dsherret/ts-morph/blob/c2e948f4055f5a19278bcba1dd4e4922da104817/src/utils/fillDefaultEditorSettings.ts
- https://github.com/dsherret/ts-morph/blob/c2e948f4055f5a19278bcba1dd4e4922da104817/src/utils/fillDefaultFormatCodeSettings.ts

As for single vs double quotes, I think we are looking for the following setting:
https://github.com/Microsoft/TypeScript/blob/97fbc87e9834b9bf12fe01bf442e16749da02d1d/lib/protocol.d.ts#L2293
```ts
readonly quotePreference?: "auto" | "double" | "single";
```

Here is a test from ts-morph that seems to be related: https://github.com/dsherret/ts-morph/blob/c2e948f4055f5a19278bcba1dd4e4922da104817/src/tests/manipulationSettingsTests.ts#L117-L123

```ts
        it("should have a single preferences after changing it", () => {
            const container = new ManipulationSettingsContainer();
            container.set({ quoteKind: QuoteKind.Single });
            doTest(container.getUserPreferences(), {
                quotePreference: "single"
            });
        });
```

Reply:
Fixed. Added code format configuration to ts-morph.